src/content/tutorials/fleet-management/cluster-management/migration-to-cluster-api/_index.md

```diff
@@ -55,3 +55,181 @@ Our engineers will check that all resources and infrastructure are correctly mig
 
 - Some customers have been using [k8s-initiator-app](https://github.com/giantswarm/k8s-initiator-app/) to configure some aspects of the workload cluster API.
   In the new Cluster API implementation, [most of the features enabled by the app](https://github.com/giantswarm/capi-migration-cli/tree/main/k8s-initiator-features) are now supported natively by the platform. The app should be removed and moved to the new syntax if our migration CLI doesn't handle your use-case. Giant Swarm account engineers will help you with this process.
+
+### Cluster manifest clean-up
+
+There are some fields in the cluster manifest that are only used during the migration, and can be cleaned up afterward. We try to make sure our migration tool cleans up the manifests and removes those fields automatically after a successful migration, but there could be some left-overs, or it could be that a cluster got migrated before that clean-up process got implemented in the tool. Bellow you'll find a non-exhaustive list of the fields that can be cleaned up (or modified) after a successful migration:
+
+- `cluster.internal.advancedConfiguration.controlPlane.etcd`: can be completely removed
+- `cluster.internal.advancedConfiguration.controlPlane.files`: everything except the following fields can be deleted
+    - To be kept
+        - `/migration/add-vintage-service-account-key.sh`
+        - `/etc/kubernetes/pki/sa-old.pem`
+- `cluster.internal.advancedConfiguration.controlPlane.preKubeadmCommands`: everything except the following fields can be deleted
+    - To be kept
+        - `/bin/sh /migration/add-vintage-service-account-key.sh`
+- `cluster.internal.advancedConfiguration.controlPlane.postKubeadmCommands`: can be completely removed
+- `internal.migration.irsaAdditionalDomain`: starting from release v25.1.1 this domain needs to be appended to `cluster.providerIntegration.controlPlane.kubeadmConfig.clusterConfiguration.apiServer.serviceAccountIssuers`, and the `internal.migration.irsaAdditionalDomain` field can be removed
+
+Here's an example manifest, and a diff with the proposed changes:
+
+```yaml
+apiVersion: v1
+data:
+  values: |
+    global:
+        metadata:
+            name: foo
+            description: example
+            organization: acme
+            servicePriority: medium
+        controlPlane:
+            instanceType: m6i.2xlarge
+        release:
+            version: 25.0.0
+        nodePools:
+            main:
+                availabilityZones:
+                    - eu-west-1a
+                instanceType: m6a.2xlarge
+                minSize: 1
+                maxSize: 9
+        providerSpecific:
+            region: eu-west-1
+    internal:
+        migration:
+            irsaAdditionalDomain: irsa.foo.k8s.bar.acme.net
```

Comment:
Let's please use `vintage` and `capi` in domain name examples to make it clear what's old vs. new

Reply:
ok, there aren't any `capi` domain names in this example, but I've used the `vintage` keyword on the vintage ones